types/gm/gm-tests.ts

```diff
@@ -101,6 +103,7 @@ gm(src)
     .colorspace(type)
     .command(customCommand)
     .compose(operator)
+    .composite(changeImage, maskImage)
```

Comment:
Could you add a test for `.composite(changeImage)`, so with the optional param left out? I see this is the case in other methods as well.